### PR TITLE
#162725226 Enable users to navigate through pagination

### PR DESCRIPTION
**Pull request by kdaglas**
#### What does this PR do?
Enables users to navigate through all the articles through different pages by using pagination.

#### Description of Task to be completed?
- add the limit for the articles for each page
- enable the number of pages required
- create a class that supports the paginated pages not to cover all the screen
- style up the numbering of the pagination
- add the navigation bar change state for when a person is logged in and not logged in

#### How should this be manually tested?
- Fetch and checkout into this branch with `git fetch origin ft-enable-pagination-162725226 && git checkout ft-enable-pagination-162725226`
- Install dependencies and start the app with `yarn install && yarn start`
- if you are not logged in, login or create an account so as to see the navigation change
- navigate to the bottom of the icons and click the next number

#### What are the relevant pivotal tracker stories?
[#162725226](https://www.pivotaltracker.com/story/show/162725226)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -12,6 +12,10 @@ import './assets/css/comments.css';
 import './assets/css/react-tags.css';
 import 'react-quill/dist/quill.snow.css';
 
+// import 'font-awesome/css/font-awesome.min.css';
+// import 'bootstrap-css-only/css/bootstrap.min.css';
+// import 'mdbreact/dist/css/mdb.css';
```
Comment (RrNn) [inline src/index.js]: Remove commented code.

File: src/assets/style.scss [hunk 2]
```diff
@@ -170,6 +172,60 @@ h4{
     text-align: center;
 }
 
+/* Pagination css */
+
+.pagination {
+    list-style: none;
+    display: inline-block;
+    padding: 0;
+    // margin-top: 10px;
```
Comment (RrNn) [inline src/assets/style.scss]: Remove this too

**Review by calvinpete — APPROVED** [review]
LGTM

**Review by MuhweziDeo — APPROVED** [review]
LGTM @kdaglas

**Pull request merged**